### PR TITLE
Update doc guide in top-level README.md and fix description of

### PR DESCRIPTION
**Pull request by davidopp**
cluster management doc in docs/admin/README.md

@bgrant0607 @gmarek

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,45 +57,46 @@ They act as basic load balancers.
 
 Kubernetes documentation is organized into several categories.
 
-  - **Getting Started Guides**
-    - for people who want to create a kubernetes cluster
-    - in [docs/getting-started-guides](docs/getting-started-guides)
-  - **User Documentation**
+  - **User documentation**
+    - for people who want to run programs on an existing Kubernetes cluster
+    - in [docs/user-guide](docs/user-guide/README.md)
```
Comment (bgrant0607) [inline README.md]: Please use doc titles or summaries, not filenames.

File: README.md [hunk 2]
```diff
@@ -57,45 +57,46 @@ They act as basic load balancers.
...
-    - in [docs/design](docs/design)
-    - for people who want to understand the design choices made
-    - describes tradeoffs, alternative designs
-    - descriptions of planned features that are too long for a github issue.
-  - **Walkthroughs and Examples**
-    - in [examples](/examples)
-    - Hands on introduction and example config files
+  - **Cluster administrator documentation**
+    - for people who want to create a Kubernetes cluster and administer it
+    - in [docs/admin](docs/admin/README.md)
```
Comment (bgrant0607) [inline README.md]: Doc titles or summaries

File: README.md [hunk 3]
```diff
@@ -57,45 +57,46 @@ They act as basic load balancers.
...
+  - **Developer and API documentation**
+    - for people who want to write programs that access the Kubernetes API, write plugins
+      or extensions, or modify the core Kubernete code
+    - in [docs/devel](docs/devel/README.md)
```
Comment (bgrant0607) [inline README.md]: Doc titles or summaries

File: README.md [hunk 4]
```diff
@@ -57,45 +57,46 @@ They act as basic load balancers.
...
+      detailed description of all fields found in the core API objects 
+  - **Walkthroughs and examples**
+    - hands-on introduction and example config files
+    - in the [user guide](docs/user-guide/README.md#quick-walkthrough)
+    - in [examples](examples/)
+  - **Contributions from the Kubernetes community**
+    - in [contrib](contrib/)
+  - **Design documentation and design proposals**
+    - for people who want to understand the design of Kubernetes, and feature proposals
+    - design docs in [docs/design](docs/design/README.md)
```
Comment (bgrant0607) [inline README.md]: Doc titles or summaries

File: README.md [hunk 5]
```diff
@@ -57,45 +57,46 @@ They act as basic load balancers.
...
+    - proposals in [docs/proposals](docs/proposals)
```
Comment (bgrant0607) [inline README.md]: Doc titles or summaries

File: README.md [hunk 6]
```diff
@@ -57,45 +57,46 @@ They act as basic load balancers.
...
+    - project roadmap in [docs/roadmap.md](docs/roadmap.md)
```
Comment (bgrant0607) [inline README.md]: Eliminate this line. The roadmap doesn't have useful info right now.

**Comment by davidopp** [comment]
Implemented your feedback, PTAL.

File: README.md [hunk 7]
```diff
@@ -57,45 +57,45 @@ They act as basic load balancers.
 
 Kubernetes documentation is organized into several categories.
 
-  - **Getting Started Guides**
```
Comment (bgrant0607) [inline README.md]: Did you intentionally delete mention of the getting-started guides?

**Comment by bgrant0607** [comment]
LGTM. Thanks.

**Comment by bgrant0607** [comment]
Please squash.

**Comment by davidopp** [comment]
Added the user guide back, PTAL

**Comment by davidopp** [comment]
And squashed

**Pull request merged**